src/components/KeepReading/KeepReading.tsx

```diff
@@ -0,0 +1,55 @@
+import { StyledKeepReading } from './StyledKeepReading';
+import { Button } from '..';
+import { useNavigate } from 'react-router';
+
+// Navigation component for How To Play section. Creates four buttons, one for each How To Play section, except the current section. No Parameters. Uses URL to identify current section.
+
+interface keepReadingProps {
+  buttons: possibleButtons[];
+}
+
+interface possibleButtons {
+  path: string;
+  title: string;
+}
+
+/**
+ * Rectangular buttons for navigation. All buttons will be displayed except the one for the page the user is currently on.
+ * @param {keepReadingProps} buttons object with a "path" property that should be the unique pathname for the desired button, without the base url. For example, "localhost:5175/how-to-play/overview" would be simple 'how-to-play/overview.' Buttons object also has a "title" property that is a simple string of the text that you want to appear in the button.
+ */
+export const KeepReading = ({ buttons }: keepReadingProps) => {
+  const url = window.location.pathname;
+  const navigate = useNavigate();
```

Comment:
Ideally this shouldn't be in a component. The component itself shouldn't care where it is in the site. Consider passing the `handleClick` function as a prop.

```suggestion
```